src/PagePropsStatementCountLookup.php

```diff
@@ -14,13 +15,15 @@ public function __construct( LoadBalancer $loadBalancer ) {
 	}
 
 	public function getStatementCount( EntityId $entityId ) {
+		$wikibaseRepo = WikibaseRepo::getDefaultInstance();
+
 		$db = $this->loadBalancer->getConnection( DB_MASTER );
 
 		$res = $db->selectRow(
 			array( 'page_props', 'page' ),
 			array( 'pp_value' ),
 			array(
-				'page_namespace' => 0,
+				'page_namespace' => array_values( $wikibaseRepo->getEntityNamespaces() ),
```

Comment:
Thanks, that makes a lot of sense. I’ve uploaded a new version that uses an `EntityNamespaceLookup`.